docs/rpc@usage/rpcclient.py

```diff
@@ -0,0 +1,60 @@
+import Pyro4
+import sys
+
+class RPCClient(object):
+    def __init__(self):
+    #Loads the variable dict with the contents of config.json.
+        self.dict = {
+                   "function-list" : {
+                            "provision" : "4",
+                            "detach_node" : "1",
+                            "create_snapshot" : "2",
+                            "list_all_images" : "1",
+                            "list_snaps" : "1",
+                            "remove_snaps" : "2"
+                            }
+                     } 
+        self.func_list = self.dict['function-list'] #The script name and no. of arguments.
+    
+    def concatenate_command(self, list_of_commands):
+        concatenated_command = ""
+        concatenated_command += " ".join(list_of_commands[1:])
+        return concatenated_command
+   
+    def correct_argument_list_length(self, list_of_commands):
+        return (len(list_of_commands)-1 <= int(self.func_list[list_of_commands[0]]))
+
+    #This method checks for escape characters or injection attacks.
+    def escape_characters_present(self, concatenated_command):
+    #The escape_chars set contains all the characters which if present would terminate the program
+    #execution  
+        escape_chars = set(';<>&|\'')
+        if any((c in escape_chars) for c in concatenated_command):
+            return True
+        else:
+            return False
+    
+    #client_function(): This function does all the check required, calls the 
+    #server program with the method name and arguments passed in as a list
+    #and prints the output received from the server.  
+    def client_function(self, list_of_commands):
+        #print(main_obj)
+       # print(self.func_list.has_key(list_of_commands[0]))
+        if (self.func_list.has_key(list_of_commands[0])):
+            concatenated_command = self.concatenate_command(list_of_commands)
+            if((not self.escape_characters_present(concatenated_command)) and self.correct_argument_list_length(list_of_commands)):
+                #print("Entering the object")
+                print main_obj.run_script(list_of_commands)
```

Comment:
Commented print statements on line 41, 42 and 46 can be removed if not needed...